mscore/musescore.cpp

```diff
@@ -246,6 +252,9 @@ const std::list<const char*> MuseScore::_basicNoteInputMenuEntries {
             "",
             "pad-rest",
             "",
+            "pad-raise",
+            "pad-lower",
+            "",
```

Comment:
Finale does have a shortcut to add an accidental the same way MuseScore does. It's just a different key (that is configurable). So these commands are not forced.